components/prism-clike.js

```diff
@@ -1,7 +1,7 @@
 Prism.languages.clike = {
 	'comment': [
 		{
-			pattern: /(^|[^\\])\/\*[\w\W]*?\*\//,
+			pattern: /(\/\*[\w\W]*?\*\/)|(\/\*[\w\W]*)/,
```

Comment:
Why not just make the closing bit optional then, with `(\/\*)?`? This is exactly what this does, but with more duplication.